sdk/keyvault/keyvault-secrets/src/index.ts

```diff
@@ -701,25 +708,25 @@ export class SecretClient {
    * Example usage:
    * ```ts
    * let client = new SecretClient(url, credentials);
-   * for await (const secretAttr of client.listSecretVersions("MySecretName")) {
+   * for await (const secretAttr of client.listPropertiesOfSecretVersions("MySecretName")) {
    *   const secret = await client.getSecret(secretAttr.name);
    *   console.log("secret version: ", secret);
    * }
    * ```
    * @param secretName Name of the secret to fetch versions for
    * @param [options] The optional parameters
    */
-  public listSecretVersions(
+  public listPropertiesOfSecretVersions(
```

Comment:
I noticed that the Python version doesn't rename this (https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/keyvault/azure-keyvault-secrets/azure/keyvault/secrets/_client.py#L192)

I think we should try to be consistent for which ones get the "properties of" treatment.

Reply:
I just talked to Isabella. They will change it!